services/management/dbaas/version_service_client.go

```diff
@@ -146,3 +154,52 @@ func (c *VersionServiceClient) Matrix(ctx context.Context, params componentsPara
 
 	return &vsResponse, nil
 }
+
+func getLatest(m map[string]componentVersion) (*goversion.Version, error) {
+	if len(m) == 0 {
+		return nil, errNoVersionsFound
+	}
+	keys := make([]*goversion.Version, len(m))
+	i := 0
+	var err error
+	for k := range m {
+		keys[i], err = goversion.NewVersion(k)
+		if err != nil {
+			return nil, err
+		}
+		i++
+	}
+
+	latest := goversion.Must(goversion.NewVersion("v0.0.0"))
+	for _, version := range keys {
+		if version.GreaterThan(latest) {
+			latest = version
+		}
+	}
+	return latest, nil
+}
+
+// GetLatestOperatorVersion return latest PXC and PSMDB operators for given PMM version.
+func (c *VersionServiceClient) GetLatestOperatorVersion(ctx context.Context, pmmVersion string) (*goversion.Version, *goversion.Version, error) {
+	if pmmVersion == "" {
+		return nil, nil, errors.New("given PMM version is empty")
+	}
+	params := componentsParams{
+		product:        "pmm-server",
+		productVersion: pmmVersion,
+	}
+	resp, err := c.Matrix(ctx, params)
+	if err != nil {
+		return nil, nil, err
+	}
+	if len(resp.Versions) != 1 {
+		return nil, nil, errors.Errorf("version service response does not cointain deps for single PMM version: length is %d", len(resp.Versions))
+	}
```

Comment:
that's possible case and we shouldn't return error. We just shouldn't return update version.

Reply:
Done